include/cling/Interpreter/InvocationOptions.h

```diff
@@ -14,16 +14,30 @@
 #include <vector>
 
 namespace cling {
+
+  class CompilerOpts {
```

Comment:
And while I'm updating this commit, any issue with making
class CIFactory -> namespace CIFactory